app/models/section.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+const { DataTypes } = require("sequelize")
+
+module.exports = (sequelize, Database) => {
+    const Section = sequelize.define('Section', {
```

Comment:
Can you add joinCode to this? It should just be a random sequence of 6 digits (probably lowercase a-z & 0-9)